modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/portlet/ClientExtensionEntryPortlet.java

```diff
@@ -194,7 +195,7 @@ private void _renderIFrame(
 				iFrameURL, (String)entry.getKey(), (String)entry.getValue());
 		}
 
-		printWriter.print(iFrameURL);
+		printWriter.print(HtmlUtil.escapeHREF(iFrameURL));
```

Comment:
:thinking: Maybe we should do this for all types and URL fields

Like for custom element, global css, global js, etc....

I'm not sure why this is possible, though, because there is a URL validation on these fields. We can forward this as is or close it and I'll investigate it under this same LPS next week (since I'm refactoring everything). Whatever you prefer...

Reply:
If you're planning on refactoring next week, I'll just let you re-use the same LPS. This was just a one-off bug I saw that thought would be easy to fix, but a more holistic solution is probably better.